BunqAPI/views.py

```diff
@@ -98,12 +99,27 @@ class MyBunqView(View):
     def get(self, request):
         user = User.objects.get(username=request.user)
         try:
-            callback(user)
+            cb = callback(user)
+            self.check_bunq_session(user, cb)
         except (ObjectDoesNotExist, KeyError):
             return HttpResponseForbidden('You are not logged in correctly.'
                                          '<a href="/account/logout">Back</a>')
         return render(request, self.template)  # pragma: no cover
 
+    @staticmethod
+    def check_bunq_session(user, cb):
+        last_login = user.last_login
+        session_end = user.session.session_end_date
+
+        print(last_login)
+        print(session_end)
+
+        if last_login <= session_end:
```

Comment:
This is not a good fix, the session will be deleted every time 🤔 This should be moved to the login view